File: Data Structures & Algorithms/duplicate-integer/submission-3.cs
public class Solution {
    public bool hasDuplicate(int[] nums) {
        HashSet<int> seen = new HashSet<int>();
        foreach (int num in nums) {
            seen.Add(num);
        }
        if (seen.Count == nums.Length) {
            return false;
        } else {
            return true;
        }
    }
}

File: Data Structures & Algorithms/sort-colors/submission-3.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        count = [0] * 3
        for num in nums:
            count[num] += 1
        i = 0
        t = 0
        while i < len(count):

            if count[i] == 0:
                i += 1
                continue
            nums[t] = i
            count[i] -= 1
            t += 1

